CHANGELOG.md

```diff
@@ -3,6 +3,229 @@
+workflows. This release contains those changes; many have been marked as breaking out of an
+abundance of caution. We expect most users to be able to upgrade without making changes.
+
+This release also includes the stabilization of a couple `uv python install` features, which have
+been available under preview since late last year.
+
+### Breaking changes
+
+- **Install Python executables into the bin ([#14626](https://github.com/astral-sh/uv/pull/14626))**
+
+  `uv python install` now installs a versioned Python executable (e.g., `python3.13`) into a
+  directory on the `PATH` (e.g., `~/.local/bin`) by default. This behavior has been available under
+  the `--preview` flag since [Oct 2024](https://github.com/astral-sh/uv/pull/8458). This change
+  should not be breaking unless it shadows a Python executable elsewhere on the `PATH`.
+
+  To install unversioned executables, e.g., `python3` and `python`, use the `--default` flag. The
+  `--default` flag has also been in preview, but is not stabilized in this release.
+
+  As with tool installation, the target directory respects common variables like `XDG_BIN_HOME` and
+  can be overridden with a `UV_PYTHON_BIN_DIR` variable.
+
+  You can opt out of this behavior with `uv python install --no-bin` or `UV_PYTHON_INSTALL_BIN=0`.
+
+  See the [documentation on installing Python
+  executables](https://docs.astral.sh/uv/concepts/python-versions/#installing-python-executables)
+  for more details.
+
+- **Register Python versions with the Windows registry ([#14625](https://github.com/astral-sh/uv/pull/14625))**
+
+  `uv python install` now registers the installed Python version with the Windows Registry as
+  specified by [PEP 514](https://peps.python.org/pep-0514/). This allows using uv installed Python
+  versions via the `py` launcher. This behavior has been available under the `--preview` flag since
+  [Jan 2025](https://github.com/astral-sh/uv/pull/10634). This change should not be breaking, as
+  using the uv Python versions with `py` requires explicit opt in.
+
+  You can opt out of this behavior with `uv python install --no-registry` or `UV_PYTHON_INSTALL_REGISTRY=0`.
+
+- **Prompt before removing an existing directory in `uv venv` ([#14309](https://github.com/astral-sh/uv/pull/14309))**
+
+  Previously, `uv venv` would remove an existing virtual environment without confirmation. While
+  this is consistent with the behavior of project commands (e.g., `uv sync`), it's surprising to users
+  that are using imperative workflows (i.e., `uv pip`). Now, `uv venv` will prompt for confirmation
+  before removing an existing virtual environment. **If not in an interactive context, uv will still
+  remove the virtual environment for backwards compatibility. However, this behavior is likely to
+  change in a future release.**
+
+  The behavior for other commands (e.g., `uv sync`) is unchanged.
+
+  You can opt out of this behavior by setting `UV_VENV_CLEAR=1` or passing the `--clear` flag.
```

Comment:
"opt out of this behavior" => something about how "opting out" means continuing to remove environments without confirmation.